packages/core/src/utils/http.ts

```diff
@@ -103,22 +103,25 @@ export async function getRelayerRaw(url: string, headers = {}) {
 
 export async function postRelayerWithAuth(
```

Comment:
kind of a nit, but i'd rather we rename this to `postWithSymmetricKeyAuth` since it is now also used for interacting with the auth server. we can create a new `postRelayerWithAuth` that calls `getSymmetricKey` and `getRelayerBaseUrl` to make those callsites a little less cluttered.